spec/car_spec.rb

```diff
@@ -37,4 +37,18 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it 'has color' do
+    car = Car.new("Ford Mustang", 1500, 36)
+
+    expect(car.color).to eq(nil)
+  end
```

Comment:
Good use of nil test first! I may have called the test something else since it doesn't have a color when initialized. `has color attribute` or `color is nil on itialization`